src/components/Results/Results.tsx

```diff
@@ -246,10 +253,11 @@ export function Results() {
     useSearchParamsState("sortBy");
```

Comment:
You also need to change the default value for sortValue here.

I'd suggest something like:
```js
const defaultSortValue = sortOptions[0].value;
const [sortValue = defaultSortValue, setSortValue] =
```

Reply:
fixed in 9ad0ac9dd64a93561f77f30864b9f35a77eeb9e0, but it was surprisingly difficult to convince typescript that `sortOptions(method)[0]` always exists. 7eeee64749e6300410e5d91f94283ebe05f55703 does so rigorously, while afe571536b694d1e982b9bf0f34d914d3e8c7203 does so with a `!` as near to `sortOptions` as possible. feel free to keep whichever one you prefer :)